assets/js/modules/thank-with-google/datastore/publications.test.js

```diff
@@ -29,9 +29,10 @@ import { MODULES_THANK_WITH_GOOGLE } from './constants';
 
 describe( 'modules/thank-with-google publications', () => {
 	let registry;
-	const PUBLICATIONS = [
+	const publicationWithActiveState = [
```

Comment:
This is much better but a bit confusing still when the variable is named as 
singular for an array. This is partly why I suggested defining individual publications here and then composing the lists in the tests.

This is ok though, I would simply ask that we always use plural names for arrays  /collections rather than named based on the knowledge of how many are in it.